src/libs/migrations/PersonalDetailsByAccountID.js

```diff
@@ -71,12 +87,17 @@ function getDeprecatedPolicyMemberListFromOnyx() {
  *     - whisperedTo -> whisperedToAccountIDs
  *     - childOldestFourEmails -> childOldestFourAccountIDs
  *     - originalMessage.participants -> originalMessage.participantAccountIDs
+ * - report_
+ *     - ownerEmail -> ownerAccountID
+ *     - managerEmail -> managerID
+ *     - lastActorEmail -> lastActorAccountID
+ *     - participants -> participantAccountIDs
```

Comment:
We are not doing such migration. We are just removing the email data and hoping the server would return the the IDs version.

Reply:
The logic here is slightly different than report actions. In reports we are blindly removing email data i.e. if a report has email data but not ids data, we will only remove the email data. The report data is now useless (no email data nor ids data). Should we delete the whole report in this case (similarly how it's done with report actions)?

Reply:
> Should we delete the whole report in this case (similarly how it's done with report actions)?

We thought about this and decided not to, because then we'd also have to delete anything related to that report (ex: report actions) so we thought it would just be better to kill the keys we're no longer using